Topics/Tech_Stacks/TinyMCE.md

```diff
@@ -0,0 +1,39 @@
+## TinyMCE API with ReactJS
+
+### What is TinyMCE API
+TinyMCE API is an free API that allows you to add an WYSIWYG(what you see is what you get) HTML editor on your website. This API essentially makes you textbox interactive for instance users can enter text, image and more in the textbox. This API is especially helpful if your app is heavy on editing or involves a lot of text changes from the user.
```

Comment:
I agree with Andrew about the fact that you should expand on this. As someone with no knowledge about TinyMCE, it is not clear to me what this API is actually used for. What does "what you see if what you get" mean? 

Maybe you could add some examples, or some links to background reading if it requires prior knowledge